docs/appendix/glossary/viam-robot-api.md

```diff
@@ -2,14 +2,15 @@
 title: Viam Robot API
 id: viam-robot-api
 full_link:
-short_description: The de facto description of how all resources can be communicated with.
+short_description: The specification for communication with resources.
 aka:
 ---
 
-The de facto description of how all resources can be communicated with.
+The specification for communication with resources.
 
-- The RDKs/SDKs implement this server side.
-- The RDKs/SDKs use this to act as clients.
-- Currently expressed as a collection of Protocol Buffer files.
-- Does not mandate gRPC as the transport mechanism.
-  - However, all RDKs/SDKs written by Viam use gRPC.
+- The RDKs/SDKs implement the Viam Robot API server-side.
```

Comment:
Is it supposed to be capitalized? Seems like it wouldn't be

Reply:
I think so. It seems to me a big enough concept to deserve capitalization. We also only seems to use this term in the SDK page